fleet/comms.go

```diff
@@ -208,6 +208,38 @@ func (svc fleetCommsService) NotifyAgentNewGroupMembership(a Agent, ag AgentGrou
 
 }
 
+func (svc fleetCommsService) NotifyAgentTags(a Agent) error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
+	defer cancel()
+
+	a, err := svc.agentRepo.RetrieveByIDWithChannel(ctx, a.MFThingID, a.MFChannelID)
+	if err != nil {
+		return err
+	}
+	agentInformation := AgentTagsRPCPayload{
```

Comment:
this can't just send a payload, it has to send RPC with AgentConfigRPCFunc in it